examples/cpp/HelloWorld.cc

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Facebook, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ */
+
+#include <cstdio>
+#include <iostream>
+#include <string>
+#include <unistd.h>
+
+#include "BPF.h"
+
+const std::string BPF_PROGRAM = R"(
+int probe_sys_clone(void *ctx) {
+  bpf_trace_printk("Hello, World!");
+  return 0;
+}
+)";
+
+int main() {
+  std::cout << "The BPF Program is: " << std::endl << BPF_PROGRAM;
+  ebpf::BPF bpf(BPF_PROGRAM);
+
+  if (!bpf.attach_kprobe("sys_clone", "probe_sys_clone")) {
+    std::cout << "Error" << std::endl;
+    return 1;
+  }
+
+  auto f = fopen("/sys/kernel/debug/tracing/trace_pipe", "r");
```

Comment:
Makes sense. This was my first `HelloWorld` Demo to make sure things work, so it is pretty primitive. Will try to improve it in the future.